packages/cli/lib/path-helpers.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
I think this file should be deleted in favour of the (dupicate? similar?) functions in `cell-operations.ts`. It's better to expose them from that package as general-purpose helpers than the cli package.